### DISCUSSION TITLE
Can't get verified: "proof has been used before" BUT NEVER USED?

### DISCUSSION THREAD

**Discussion by mongfanya** [community/community#121033] [GitHub Education] [answered]
### Hi everyone,

Hi,

I just tried applying to GitHub education and my application has been rejected because the document I used for proof has somehow been used before? I don't understand how this is happening since I have a rare name and I haven't used in any other account.

What should I do?

Mong--Valenciano Alejandro

**Comment by eliz-amst** [comment, marked as answer]
Have you tried to apply before? or load a file for the application? It simply means that you tried to apply before and uploaded this file(your proof) and it was rejected, simply reapply but this time submit another file, not this one since it was not approved, try submitting your student ID, or transcript, make sure there is your full name displayed same as in your GitHub account.

**Reply by mongfanya** [reply]
Ohhh, I did get a notification that I had to add 2FA and my name in the billing information before I could get approved. Thanks! I guess I'll have to wait for a transcript cause it said my id wasn't enough evidence (no photo on the front i guess) and I used my acceptance letter the first time.

Thanks for the explanation!
